README.rst

```diff
@@ -17,7 +17,7 @@ Install from source:
 
     $ git clone https://github.com/tumblr/pytumblr.git
     $ cd pytumblr
-    $ python setup.py install
+    $ python -m build
```

Comment:
Do we need to update `.github/workflows/ci.yaml` as well?

Reply:
The PR build wasn't working for you because the testing steps didn't include installing `build`. I updated the instructions and `ci.yaml` to include installing and using `build`.